src/jdk.internal.vm.ci/share/classes/jdk.vm.ci.meta/src/jdk/vm/ci/meta/MetaUtil.java

```diff
@@ -50,7 +50,7 @@ public static String getSimpleName(Class<?> clazz, boolean withEnclosingClass) {
             }
             return simpleName;
         }
-        // Must be an anonymous or local class
+        // Must be a local class
```

Comment:
This file should not be changed.  It refers to the Java language anonymous class (not VM anonymous class).

Reply:
Changes have been restored.